csharp/private/actions/assembly.bzl

```diff
@@ -83,12 +83,13 @@ def AssemblyAction(
     if debug:
         args.add("/debug+")
         args.add("/optimize-")
+        args.add("/define:TRACE;DEBUG")
```

Comment:
Aren't these defined in the `csproj`?

Reply:
In modern style csproj it’s implict. So, yes and no.

Reply:
Hmm okay. Fine for now then I guess.

I was sort of thinking people should just define them on their rules. Is there a way to _undefine_ them for a dotnet csproj?

Reply:
Probably via the update/remove mechanism. I support adding this to our rules too!

We talked IRL about how we'll probably want a `stdlib = False` flag on our rules once we auto-add the stdlib for similar reasons.